app/models/manageiq/providers/google/inventory/collector.rb

```diff
@@ -6,40 +6,56 @@ def initialize(_manager, _target)
     super
   end
 
-  def connection
-    @connection ||= manager.connect
+  def compute
+    @compute ||= manager.connect(:service => "compute")
+  end
+
+  def sql
+    @sql ||= manager.connect(:service => "sql")
   end
 
   attr_reader :project_key_pairs
 
   def availability_zones
-    connection.zones.all
+    compute.zones.all
+  end
+
+  def cloud_databases
+    @cloud_databases ||= sql.instances.all
+  rescue Google::Apis::ClientError
```

Comment:
If you don't enable the sqladmin API GCE raises a ClientError exception.  This catches and ignores it in case the user doesn't care about sql databases